doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+
+:::{admonition} **Note 1: Black Box Definition**
+The use of Black Box here varies subtly from the conventional use within machine learning. Typically, we say a model is a black box if the mechanism by which it makes predictions is too complicated to be interpretable to a human. Here we use black-box to mean that the explainer method doesn't need access to the model internals to be applied.
+:::
+
+## Applications:
+
+**Trust:** At a core level, explainability builds trust in the machine learning systems we use. It allows us to justify their use in many contexts where an understanding of the basis of the decision is paramount.
+
+**Testing:** Explainability can be thought of as an extra form of testing for a model. The insights derived should conform to the expected behaviour. Failure to do so may indicate issues with the model or problems with the dataset it's been trained on.
+
+**Functionality:** Insights can be used to augment model functionality. For instance, providing information on top of model predictions such as how to change model inputs to obtain desired outputs.
+
+**Research:** Explainability allows researchers to understand how and why opaque models make decisions. Helping them understand more broadly the effects of the particular model or training schema they're using.
+
+__TODO__:
+- picture of explainability pipeline: training -> prediction -> insight
+
+:::{admonition} **Note 2: Biases**
+
+Practitioners must be wary of using explainability to excuse incorrect models rather than ensuring their correctness. It is possible to have a correctly trained model on a dataset, but the model doesn't reflect reality because the dataset is either wrong or incomplete. Suppose the insights that explainability generates conform to some confirmation bias of the person training the model. In that case, they will be blind to this issue and instead use these methods to confirm erroneous results. A key distinction is that explainability insights are designed to be faithful to the model they are explaining and not the data. You use an explanation to obtain an insight into the data only if the model is trained well.
+:::
+
+## Global and Local Insights
+
+Insights can be categorised into two types. Local and global. Intuitively, a local insight says something about a single prediction that a model makes. For example, given an image classified as a cat by a model, a local insight might give the set of features (pixels) that need to stay the same for that image to remain classified as a cat. Such an insight provides an idea of what the model is looking for when classifying a specific instance into a particular class.
```

Comment:
"Local and global" shouldn't be a standalone sentence, I'd suggest merging with the previous sentence with an em-dash: —